src/js/actions.js

```diff
@@ -78,15 +77,14 @@ export function setViewedPages(pageKeys) {
   };
 }
 
-function submitToUrl(body, submitUrl, trackingPrefix) {
+function submitToUrl(body, submitUrl, recordEvent) {
   return new Promise((resolve, reject) => {
     const req = new XMLHttpRequest();
     req.open('POST', submitUrl);
     req.addEventListener('load', () => {
       if (req.status >= 200 && req.status < 300) {
-        recordEvent({
-          event: `${trackingPrefix}-submission-successful`,
-        });
+        recordEvent({ event: 'form-submit-successful' });
```

Comment:
The function that gets called is provided by the app (in that case, the vets.gov website) so they can make whatever changes they'd like for each event, and it can be customized per event. We don't need to incorporate any values in the `formConfig` since they can add whatever they want in their function. One thing that I did which may be worth reconsidering is I used `.bind()` to bind the function to the `formConfig`, but on second thought it makes more sense to have the creator of `form.js` do that if that's what they want. It's possible they might want to pass in a bound "EventReportingClass" method that encapsulates whatever data they need including `trackingPrefix`.